Docs Updates to Follow Style Guides. (#12924)
hashicorp/vagrant: website/content/docs/installation/index.mdx

---
layout: docs
page_title: Install Vagrant
description: |-
  Vagrant is available for most platforms. Install the Vagrant
  package using standard procedures for your operating system.
---

# Install Vagrant

> To get started with Vagrant, you will need to download the appropriate installer or

Reviewer: ```suggestion
To get started with Vagrant, download the appropriate installer or
```

package for your platform from our
[Vagrant downloads page](/downloads). Install the package with the standard procedures for
your operating system.

> The installer should automatically add `vagrant` to your system path

Reviewer: ```suggestion
The installer automatically adds `vagrant` to your system path
```

so that it is available in terminals. If it is not found,
log out and back into your system; this is a common issue for Windows.

~> **Rubygem installation is unsupported** Vagrant 1.0.x has the option to
be installed as a [RubyGem](https://en.wikipedia.org/wiki/RubyGems).
However, this installation method is no longer supported. If you have an old version
of Vagrant installed via Rubygems, remove it prior to installing newer
versions of Vagrant.

## First development environment

If you are new to Vagrant, the next step to set up a development environment is to install
a [box](https://developer.hashicorp.com/vagrant/tutorials/getting-started/getting-started-boxes).

## How to use multiple hypervisors

> If you have more than one hypervisor is in use, hypervisors often do not allow you to bring up virtual machines. 

Reviewer: ```suggestion
Hypervisors often do not allow you to bring up virtual machines if you have more than one hypervisor in use.
```

VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

> You must denylist the additional hypervisors in order for VirtualBox to run correctly.

Reviewer: ```suggestion
You must add the additional hypervisors to the deny list in order for VirtualBox to run correctly.
```
`Denylist` as a verb sounds odd to me but feel free to keep it as I learned today that it is, in fact, a verb 😅

irqbypass              16384  1 kvm
```

> Use the `blacklist` command to add the hypervisor to you denylist.

Reviewer: ```suggestion
Use the `blacklist` command to add the hypervisor to your denylist.
```
